webdriver/tests/element_send_keys/send_keys.py

```diff
@@ -71,3 +71,41 @@ def test_stale_element(session, inline):
 
     response = element_send_keys(session, element, "foo")
     assert_error(response, "stale element reference")
+
+@pytest.mark.parametrize("value",[
+    [u'\ue018', ";"],
```

Comment:
What I didn't notice before I actually run these tests locally was that for parameterization you have to use tuples. At least for me pytest complains about that. Also it avoids to have to call the indices. 

```suggestion
@pytest.mark.parametrize("value, expected",[
    (u'\ue018', ";"),
```

Reply:
updated